extensions/shared/icons.js

```diff
@@ -90,3 +95,69 @@ export const JetpackLogo = ( { size = 24, className } ) => (
 		<Polygon className="jetpack-logo__icon-triangle" fill="#fff" points="17,29 17,13 25,13 " />
 	</SVG>
 );
+
+// @TODO: Import those from https://github.com/Automattic/social-logos when that's possible.
```

Comment:
Is there an Issue for this problem that we can follow up on?

Reply:
This code including the TODO was moved from elsewhere in the codebase and I don't think anything has changed around the ways it could be more cleverly incorporated.